tests/validation/converters/test_vision_excel_converter.py

```diff
@@ -332,11 +337,11 @@ def test_guid_extra_info():
     vision_cvtr = prep_vision_97(language=LANGUAGE_EN)
     _, extra_info = vision_cvtr.load_input_data()
 
-    assert extra_info[0]["GUID"] == "{8110C18E-7FDF-4A43-B868-E05E1DD7909F}"
-    assert extra_info[1]["GUID"] == "{D507B771-AC81-41F5-A292-91CF6119BE70}"
-    assert extra_info[2]["GUID"] == "{C92556BA-57E6-4780-A69E-DDDCA759BFBE}"
-    assert extra_info[3]["GUID"] == "{23C17CB9-7C1F-4AD0-B3C9-9B4AB99DB400}"
-    assert extra_info[4]["GUID"] == "{915748FF-2EE0-4C5E-806A-DBBFC433BEA4}"
+    assert extra_info[0]["GUID"] == "{7FF722ED-33B3-4761-84AC-A164310D3C86}"
```

Comment:
Should not have needed this. But somehow, a fresh export with this file made new GUIDs in the excel. Maybe because the vnf files version was changed. 
Maybe it originated because vision file from v9.4 to v9.7.

Something to keep in mind. @Jerry-Jinfeng-Guo

Reply:
They are abusing UUID! We will run out of them sooner I'm afraid